bodhi/server/consumers/masher.py

```diff
@@ -1006,3 +1037,24 @@ def run(self):
         else:
             self.success = True
         return out, err, returncode
+
+def system(cmd):
+    """
+    Invoke a shell command.
+    :returns: A tuple of output, err message, and return code
+    """
+    ret = subprocess.Popen(cmd, shell=True, stdin=subprocess.PIPE,
+                stdout=subprocess.PIPE, stderr=subprocess.PIPE, close_fds=True)
+    out, err = ret.communicate()
+    return out, err, ret.returncode
```

Comment:
I think there's already a function to do something like this in ```bodhi.server.util.cmd()```, and I think it is used a few places here in ```masher.py```. I recommend consolidating around the existing one rather than creating a new one.